File: salario.java
import java.util.Scanner;

public class salario {
    public static void main(String[] args) {
        // Ler valores de entrada
        Scanner leitorDeEntradas = new Scanner(System.in);
        float valorSalario = leitorDeEntradas.nextFloat();
        float valorBeneficios = leitorDeEntradas.nextFloat();

        float valorImposto = 0;
        if (valorSalario >= 0 && valorSalario <= 1100) {
            // Aplicar aliquota de 5%
            valorImposto = valorSalario * 0.05F;
        } else if (valorSalario > 1100 && valorSalario <= 2500) {
            valorImposto = valorSalario * 0.1F;
        } else {
            valorImposto = valorSalario * 0.15F;
        }

        float saida = valorSalario - valorImposto + valorBeneficios;
        System.out.println(String.format("%.2f", saida));
    }
}

File: salario.js
// gets e print são funções implementadas pela DIO

const valorSalario = parseFloat(gets());
const valorBeneficios = parseFloat(gets());

const valorImposto = calcularImposto(valorSalario);
const saida = valorSalario - valorImposto + valorBeneficios;
println(saida.toFixed(2));

function calcularImposto(salario) {
    let aliquota = 0;
    if (salario >= 0 && salario <= 1100) {
        aliquota = 0.05;
    } else if (salario > 1100 && salario <= 2500) {
        aliquota = 0.1;
    } else {
        aliquota = 0.15;
    }
    return aliquota * salario;
}